# This rule ensures that all import statements within a module of a feature-sliced design adhere to the architecture&#39;s guidelines. It validates that imports are relative and contained within their respective feature modules to maintain modular integrity. The rule prevents imports that reach out of the module&#39;s scope, reinforcing tight coupling within features and loose coupling between them.&#34; (`check-imports`)

Please describe the origin of the rule here.

## Rule Details

This rule aims to...

Examples of **incorrect** code for this rule:

```js

// fill me in

```

Examples of **correct** code for this rule:

```js

// fill me in

```

### Options

If there are any options, describe them here. Otherwise, delete this section.

## When Not To Use It

Give a short description of when it would be appropriate to turn off this rule.

## Further Reading

If there are other links that describe the issue this rule addresses, please include them here in a bulleted list.
